app-crypt/zulucrypt/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST zulucrypt-5.1.0.tar.bz2 1610597 BLAKE2B 3b77676912f0cd8990ba307ff04309f308ec29cf76cad37ac28ebb317a7e42ee93ffe9deb1db12a80c4d5b235c2a83bc8e5ddeb6082e5f1be9d91425d06f4334 SHA512 b916d43f8c889f1593c506d95360b195f0639024463eb77b5b21b544468f99719b877d8c94725504305f307efe5cb18e203dc8f9b015bd55825dc1d4fffb42f0
```

Comment:
Please use a 'short' URL in `Closes` tag. Also, please try to put useful bug desc in summary line instead of useless number (if you can fit it).

Reply:
How about `app-crypt/zulucrypt: Fix build with USE=-qt5` - Bug url should be `https://bugs.gentoo.org/647264`

Reply:
changed commit message